### ISSUE TITLE
Failed contraction

### ISSUE THREAD

**Issue by NickHu** [homotopy-io/homotopy-rs#899]
1. Compose algebraic cap with a real cup to make a bubble;
2. Take an endomorphism on it (left-hand side of screenshot);
3. Compose with bubble (right-side side of screenshot);
4. Contract

Contraction fails with unconnected max dimensional generator

![image](https://user-images.githubusercontent.com/450276/201182829-3568fc7a-c6f1-40bc-b321-b447b847ce2a.png)
![image](https://user-images.githubusercontent.com/450276/201182950-dc5d4f2b-b5db-4aa6-9239-3d852a3b4da9.png)

This doesn't happen if the cup is algebraic also, and the contraction succeeds.

**Comment by NickHu** [comment]
While the contraction succeeds when an algebraic cup is used, I can reproduce a crash instead:
1. ![image](https://user-images.githubusercontent.com/450276/201183519-4f7241d4-50d3-4014-bc24-b6e963b77908.png)
2. ![image](https://user-images.githubusercontent.com/450276/201183574-0ea8ba21-a68c-4574-a3ca-80156ac350f8.png)
3. ![image](https://user-images.githubusercontent.com/450276/201183621-2625fa66-b866-47ad-b609-5a2b7b95762e.png)

Panic on 2 -> 3:

```
panicked at 'range end index 2 out of range for slice of length 0', homotopy-core/src/diagram.rs:390:17

Stack:

getImports/imports.wbg.__wbg_new_abda76e883ba8a5f@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:429:21
console_error_panic_hook::hook::h71f6aa5043085852@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[2043]:0x2c3c91
core::ops::function::Fn::call::h5f6b1ca6e90215d5@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[5398]:0x35a86a
std::panicking::rust_panic_with_hook::h70a0e195f4db2a29@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[3495]:0x333b3c
std::panicking::begin_panic_handler::{{closure}}::hdcfc819ce836829e@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[3796]:0x3418f9
std::sys_common::backtrace::__rust_end_short_backtrace::h53cabafab5b09ada@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[4638]:0x355faf
rust_begin_unwind@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[4309]:0x350892
core::panicking::panic_fmt::h751be80779d42b53@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[4544]:0x354c8f
core::slice::index::slice_end_index_len_fail_rt::hfcf93dd35f0112bd@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[4081]:0x34aeb8
core::slice::index::slice_end_index_len_fail::h3db476b0d09994d2@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[5186]:0x359d8a
homotopy_core::diagram::DiagramN::rewrite_forward::h8b89d9399b4a4fb0@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[373]:0x12394c
homotopy_core::diagram::Diagram::rewrite_forward::h67c4d64d415c9a80@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1855]:0x2ae875
homotopy_core::check::<impl homotopy_core::diagram::DiagramN>::check_worker::h574106099025bfda@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[584]:0x19cc36
homotopy_core::diagram::DiagramN::new::hbade2cc08da11777@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1742]:0x2a0a8f
homotopy_core::contraction::colimit::h47f5934355fde95f@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[230]:0x2e9c2
homotopy_core::contraction::contract_base::h75cc99ce9ae37092@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[284]:0xbb463
homotopy_core::contraction::<impl homotopy_core::diagram::DiagramN>::contract::{{closure}}::h7d21c53f39a72272@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[447]:0x158674
homotopy_core::attach::attach_worker::hdea0261b3ed694c1@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[503]:0x176cf3
homotopy_model::proof::ProofState::homotopy_contraction::h6ff060eda623aa58@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[491]:0x170e07
homotopy_model::proof::ProofState::update::hc34cd39c9bb6b583@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[232]:0x37467
homotopy_web::model::State::update::hba38d826059132f2@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[259]:0x8a46b
<homotopy_web::app::App as yew::html::component::Component>::update::hb8d6c8731f3110ba@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[458]:0x15ee68
<yew::html::component::lifecycle::UpdateRunner<COMP> as yew::scheduler::Runnable>::run::h64a67877a10d8697@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[723]:0x1d1634
std::thread::local::LocalKey<T>::with::h83f62ecd67e24327@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1997]:0x2bede5
yew::scheduler::start::h4125fb360dc06f79@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[5562]:0x35ae6a
<homotopy_web::app::diagram_svg::DiagramSvg<_> as yew::html::component::Component>::view::{{closure}}::ha0fa6cd0dd4d4bc5@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[3618]:0x339b41
yew::callback::Callback<IN>::emit::h0b06e662892a975d@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[3566]:0x337346
<yew::html::listener::events::onclick::Wrapper as yew::virtual_dom::listeners::Listener>::handle::hb6106d7c3c5308ac@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[5455]:0x35aa9c
yew::virtual_dom::listeners::Registry::run_handlers::{{closure}}::hffdf538b23b58447@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1408]:0x2709fd
yew::virtual_dom::listeners::Registry::handle::h8e2decb4cd0db96b@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[2482]:0x2ef3c7
yew::virtual_dom::listeners::GlobalHandlers::ensure_handled::{{closure}}::{{closure}}::hbe2378ca4bc43155@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[5125]:0x3599f4
<dyn core::ops::function::Fn<(A,)>+Output = R as wasm_bindgen::closure::WasmClosure>::describe::invoke::hd28ec96e620d218e@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[4864]:0x3581c4
__wbg_adapter_51@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:273:10
real@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:202:20
EventListener.handleEvent*EventTarget.prototype.addEventListener</<@eval:4:42
getImports/imports.wbg.__wbg_addEventListener_1fc744729ac6dc27/<@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:686:25
handleError@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:285:18
getImports/imports.wbg.__wbg_addEventListener_1fc744729ac6dc27@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:685:79
web_sys::features::gen_EventTarget::EventTarget::add_event_listener_with_callback_and_add_event_listener_options::h1e81f2971d403074@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[4422]:0x352c5c
std::thread::local::LocalKey<T>::with::h94ccdd0b2030b966@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1833]:0x2abdfc
yew::virtual_dom::listeners::Registry::register::h45a2baf6ce6aa9f6@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[626]:0x1adee4
std::thread::local::LocalKey<T>::with::hf0a365005e48027d@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[3544]:0x33628e
<yew::virtual_dom::listeners::Listeners as yew::virtual_dom::Apply>::apply::h73ff317f149ae175@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[2745]:0x3049ef
<yew::virtual_dom::vtag::VTag as yew::virtual_dom::VDiff>::apply::h96d7bb5f77d983d1@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[395]:0x136626
<yew::virtual_dom::vnode::VNode as yew::virtual_dom::VDiff>::apply::hb94a3da82243c9f1@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[679]:0x1c1dc7
<yew::html::component::lifecycle::RenderRunner<COMP> as yew::scheduler::Runnable>::run::hd60eca61f7c775ed@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1118]:0x23ab5d
std::thread::local::LocalKey<T>::with::h83f62ecd67e24327@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1997]:0x2bede5
yew::scheduler::start::h4125fb360dc06f79@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[5562]:0x35ae6a
yew::html::component::scope::Scope<COMP>::callback_with_passive::{{closure}}::h29d15cd2f72f2bd3@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[4291]:0x350249
yew::callback::Callback<IN>::emit::h4eebdf2be889b3f5@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[3496]:0x333bb2
gloo_file::file_reader::callbacks::read_as_bytes::{{closure}}::h18728d4c3c70fdee@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1590]:0x28c595
gloo_file::file_reader::callbacks::read::{{closure}}::h65482b1f0e04a141@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[3113]:0x31e618
<dyn core::ops::function::FnMut<(&A,)>+Output = R as wasm_bindgen::closure::WasmClosure>::describe::invoke::h98b5086eb9e9abba@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[4518]:0x35463c
__wbg_adapter_44@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:266:14
real@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:229:20
EventListener.handleEvent*EventTarget.prototype.addEventListener</<@eval:4:42
getImports/imports.wbg.__wbg_addEventListener_1fc744729ac6dc27/<@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:686:25
handleError@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:285:18
getImports/imports.wbg.__wbg_addEventListener_1fc744729ac6dc27@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:685:79
web_sys::features::gen_EventTarget::EventTarget::add_event_listener_with_callback_and_add_event_listener_options::h1e81f2971d403074@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[4422]:0x352c5c
std::thread::local::LocalKey<T>::with::h147763118591b09e@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[2821]:0x30a7d0
gloo_file::file_reader::callbacks::read_as_bytes::h6584d3244ffe7b3f@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1939]:0x2b8609
<homotopy_web::app::project::ProjectView as yew::html::component::Component>::update::hc38b90498e23c9ad@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1040]:0x229179
<yew::html::component::lifecycle::UpdateRunner<COMP> as yew::scheduler::Runnable>::run::hdf6cab7c82f5759b@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[505]:0x178235
std::thread::local::LocalKey<T>::with::h83f62ecd67e24327@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1997]:0x2bede5
yew::scheduler::start::h4125fb360dc06f79@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[5562]:0x35ae6a
yew::html::component::scope::Scope<COMP>::callback_with_passive::{{closure}}::h760e2dc0cb7963c9@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[2831]:0x30b4a2
yew::callback::Callback<IN>::emit::h0b06e662892a975d@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[3566]:0x337346
<yew::html::listener::events::oncut::Wrapper as yew::virtual_dom::listeners::Listener>::handle::h236bb2e8813c8394@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[5462]:0x35aae2
yew::virtual_dom::listeners::Registry::run_handlers::{{closure}}::hffdf538b23b58447@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1408]:0x2709fd
yew::virtual_dom::listeners::Registry::handle::h8e2decb4cd0db96b@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[2482]:0x2ef355
yew::virtual_dom::listeners::GlobalHandlers::ensure_handled::{{closure}}::{{closure}}::hbe2378ca4bc43155@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[5125]:0x3599f4
<dyn core::ops::function::Fn<(A,)>+Output = R as wasm_bindgen::closure::WasmClosure>::describe::invoke::hd28ec96e620d218e@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[4864]:0x3581c4
__wbg_adapter_51@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:273:10
real@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:202:20
EventListener.handleEvent*EventTarget.prototype.addEventListener</<@eval:4:42
getImports/imports.wbg.__wbg_addEventListener_1fc744729ac6dc27/<@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:686:25
handleError@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:285:18
getImports/imports.wbg.__wbg_addEventListener_1fc744729ac6dc27@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:685:79
web_sys::features::gen_EventTarget::EventTarget::add_event_listener_with_callback_and_add_event_listener_options::h1e81f2971d403074@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[4422]:0x352c5c
std::thread::local::LocalKey<T>::with::h94ccdd0b2030b966@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1833]:0x2abdfc
yew::virtual_dom::listeners::Registry::register::h45a2baf6ce6aa9f6@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[626]:0x1adee4
std::thread::local::LocalKey<T>::with::hf0a365005e48027d@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[3544]:0x33628e
<yew::virtual_dom::listeners::Listeners as yew::virtual_dom::Apply>::apply::h73ff317f149ae175@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[2745]:0x3049ef
<yew::virtual_dom::vtag::VTag as yew::virtual_dom::VDiff>::apply::h96d7bb5f77d983d1@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[395]:0x136626
<yew::virtual_dom::vnode::VNode as yew::virtual_dom::VDiff>::apply::hb94a3da82243c9f1@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[679]:0x1c1dc7
yew::virtual_dom::vlist::VList::apply_unkeyed::h92ca0cc010b0d32b@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1207]:0x24cf40
<yew::virtual_dom::vlist::VList as yew::virtual_dom::VDiff>::apply::heba9a2e645f12a54@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[279]:0xb2cb7
<yew::virtual_dom::vnode::VNode as yew::virtual_dom::VDiff>::apply::hb94a3da82243c9f1@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[679]:0x1c1f4a
<yew::html::component::lifecycle::RenderRunner<COMP> as yew::scheduler::Runnable>::run::he9d3be3f89b05d74@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1120]:0x23b258
std::thread::local::LocalKey<T>::with::h83f62ecd67e24327@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1997]:0x2bede5
yew::scheduler::start::h4125fb360dc06f79@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[5562]:0x35ae6a
yew::html::component::scope::Scope<COMP>::callback_with_passive::{{closure}}::h37170eb2c75db4c9@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[4292]:0x35029f
yew::callback::Callback<IN>::emit::h4eebdf2be889b3f5@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[3496]:0x333bb2
yew::callback::Callback<IN>::reform::{{closure}}::h08bc1d8a98d619df@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[3697]:0x33d653
yew::callback::Callback<IN>::emit::h0b06e662892a975d@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[3566]:0x337346
<yew::html::listener::events::onclick::Wrapper as yew::virtual_dom::listeners::Listener>::handle::hb6106d7c3c5308ac@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[5455]:0x35aa9c
yew::virtual_dom::listeners::Registry::run_handlers::{{closure}}::hffdf538b23b58447@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[1408]:0x2709fd
yew::virtual_dom::listeners::Registry::handle::h8e2decb4cd0db96b@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[2482]:0x2ef355
yew::virtual_dom::listeners::GlobalHandlers::ensure_handled::{{closure}}::{{closure}}::hbe2378ca4bc43155@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[5125]:0x3599f4
<dyn core::ops::function::Fn<(A,)>+Output = R as wasm_bindgen::closure::WasmClosure>::describe::invoke::hd28ec96e620d218e@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web_bg.wasm:wasm-function[4864]:0x3581c4
__wbg_adapter_51@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:273:10
real@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:202:20
EventListener.handleEvent*EventTarget.prototype.addEventListener</<@eval:4:42
getImports/imports.wbg.__wbg_addEventListener_1fc744729ac6dc27/<@https://homotopy-rs--homotopy-io-labelled-xvbfeqnk.web.app/homotopy_web.js:686:25


homotopy_web.js:441:21
```

**Comment by NickHu** [comment]
Fixed by 70ef303e489f489d06c982981dd0e2487efb6667
